gitignore

```diff
@@ -1 +1,2 @@
 *.txt
```

Comment:
ah, the name of the file should start with a dot '.'.  Thus when you do an 'ls' you don't see it, but git will still use it.  This is common for configuration files.  This way they don't clutter the view when you do an ls.

But more importantly in this case, git won't recognize this unless it starts with a dot